src/StateLabel.js

```diff
@@ -2,17 +2,18 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import {GitMerge, GitPullRequest, IssueClosed, IssueOpened} from '@primer/octicons-react'
+import {variant} from 'styled-system'
 import theme, {colors} from './theme'
 import {COMMON, get} from './constants'
 import StyledOcticon from './StyledOcticon'
 
 const statusMap = {
-  issueClosed: colors.red[6],
-  pullClosed: colors.red[5],
-  pullMerged: colors.purple[5],
+  issueClosed: get('colors.red.6'),
+  pullClosed: get('colors.red.5'),
+  pullMerged: get('colors.purple.5'),
```

Comment:
Quick note as I'm passing by! We should double check that this will respect a custom theme passed in via props. this might need to be:
issueClosed: get('colors.red.6')(props)

Reply:
@emplums You're correct; I have a few proposed changes in a branch but haven't gotten back to this yet